### ISSUE TITLE
Static resources for custom app broken

### ISSUE THREAD

**Issue by JedWatson** [keystonejs/keystone#1369]
An unintentional compatibility-breaking change was introduced in 067caf68264eb0be3e0df7e5980a81aa119f9b68, because `keystone.static` was replaced with the new Admin App Static Router.

A method should be exposed as `Keystone.prototype.static` which binds this and restores the previous functionality until `0.4.0` when this will be released as a breaking change.

**Comment by JedWatson** [comment]
I've also exposed `keystone.adminApp.staticRouter` which will be the new way to include the Admin UI in a custom express app config. It should be invoked like:

```
app.use('/keystone', keystone.adminApp.staticRouter);
```

... and should ideally be placed above any dynamic middleware (session management, body parsing, etc.)

Note that it _must_ be mounted at `/keystone` or paths in the CSS and client-side JS will break!

**Comment by v1p** [comment]
Awesome! Thanks a ton for this!

**Comment by JedWatson** [comment]
No problem, sorry I broke it in the first place! Forgot people would be using it this way, I actually refrained from refactoring `.mount` before we get closer to 0.4.0 for this reason.

Will release a new Keystone version including this fix shortly.

**Comment by v1p** [comment]
No worries mate! We'll need to update wiki page too with the new release, as that's the only guide at the moment to perform this integration & it works flawlessly !

https://github.com/keystonejs/keystone/wiki/How-to-Add-Keystone-to-an-Already-Existing-Express-App

**Comment by morenoh149** [comment]
@v1p please do so. I wrote most of that guide and now you have exceeded my knowledge for this use case.

**Comment by BerkeleyTrue** [comment]
Currently breaking app. When will this be released?

**Comment by BerkeleyTrue** [comment]
Quick fix for those who need it: 

use
```js
var adminAppRouter = require('keystone/admin/app/static');

app.use('/keystone', adminAppRouter);

// instead of

keystone.static(app);

```

@JedWatson Should I update the wiki entry above with this?

**Comment by v1p** [comment]
@morenoh149 my bad, I was away and couldn't update the guide. @BerkeleyTrue, please do so, thanks a lot.

**Comment by BerkeleyTrue** [comment]
As soon as jed chimes in. I am not sure if keystone.static is being re-introduced as a permanent api or deprecated fix in favor of `app.use('/keystone', keystone.adminApp.staticRouter)` (which has not landed yet but looks more legit to me). The fix I posted above seems like a hack.

**Comment by v1p** [comment]
@BerkeleyTrue Sure, Jed has pushed the fix to master already I guess. I am still away & have intermittent access to internet & wasn't able to test the given fix.

**Comment by JedWatson** [comment]
`keystone.static` is back for compatibility, but deprecated. `app.use('/keystone', keystone.adminApp.staticRouter)` is the way to go, there will be more like this. I'll make a list of changes for 0.4.

**Comment by BerkeleyTrue** [comment]
Will make changes to wiki accordingly.

**Comment by JedWatson** [comment]
Thanks @BerkeleyTrue!

**Comment by BerkeleyTrue** [comment]
Updated with deprecation note and hack for 0.3.8. I will update when fix lands in 0.3.9 and remove hack.

**Comment by BerkeleyTrue** [comment]
Removed reference of hack in wiki and updated to point users to the new staticRouter property.
